tshirts.c

```diff
@@ -14,6 +14,7 @@ char size(int cms) {
 }
 
 int main() {
+    assert(size(38) == 'S' ||size(38) == 'M');

```

Comment:
Do you expect size 38 to be S or M?
you need to clearly state the expected behavior (whatever you assume) using asserts and cover all range of values with boundary values.

Reply:
Yes, as 38 is at boundary so covered both S and M for equivalence check

Reply:
Size 38 can be either S or M ,can't be both. Whichever you assume should be stated using asserts.

Reply:
Done!